condor_overview2.py

```diff
@@ -0,0 +1,365 @@
+#!/usr/bin/env python
+"""
+    Condor Overview:
+    Script to summarize output from condorq,
+    Shows:
+    - a table of running jobs: type of job-task vs. site.
+    - a table of pending jobs: type of job-task vs. site
+    - jobs that have run for more than 24 hours
+    - jobs that have restarted more than 3 times
+    
+"""
+import sys
+import optparse
+from random import choice
+try:
+    import htcondor
+except ImportError:
+    print "You do not have a proper environment, please source the following:"
+    print "source /data/srv/wmagent/current/apps/wmagent/etc/profile.d/init.sh"
+    sys.exit(1)
+
+global_pool = 'vocms099.cern.ch'
```

Comment:
I changed the line Paola mentioned above, so that global_pool points to the landb alias 'cmsgwms-collector-global.cern.ch' for monitoring systems. I've tested it and it works as expected.